### DISCUSSION TITLE
docker开启的Alist在改全局内容报错为Request failed with status code 403

### DISCUSSION THREAD

**Discussion by XEYt44** [AlistGo/alist#6540] [Invalid question]
服务器用docker开启的alist刚开启的时候正常，就是更新索引后就这样了在其他系统开启就没问题了，版本为:v3.35.0我在Windows和Linux本地开启过alist通过恢复文件运行缺没问题，服务器的nginx也是正常，索引更新报错为
![image](https://github.com/alist-org/alist/assets/121798130/307ae471-9dc7-4b81-9de4-7070c40546de)
修改全局报错为
![image](https://github.com/alist-org/alist/assets/121798130/44728bd7-53a1-4ead-9509-bd15abbce9ed)

这个代码报错我找到的都是云盘挂载失败的案例我确定看了历史的issues和官方文档但没发现有我这种类似的问题

**Comment by JerryLYU233** [comment]
我也是，需要alist拉取背景之类的都报错

**Reply by XEYt44** [reply]
换Linux了就没这问题了

**Comment by wanyunr** [comment]
同样遇到该问题，但是我是使用linu下二进制文件安装的，使用另一台机器反代的，IP访问没有问题，反代网站的日志也没有`error`，经过排查，发现是1panel的防火墙的问题

**Reply by mohan9898** [reply]
这个解决了吗

**Reply by wanyunr** [reply]
> 这个解决了吗

在1panel上给请求路径加白名单，或者给常用ip加白名单

**Reply by XEYt44** [reply]
建议用Linux版本的流畅度和兼容Linux都很好

**Comment by Yinhono** [comment]
同问题+1，我是1panel反代建的，防火墙关了也没用，还在查

**Reply by mohan9898** [reply]
不用关防火墙，把waf全局关了就可以了

**Reply by Yinhono** [reply]
> 不用关防火墙，把waf全局关了就可以了

感谢，已解决
